swupd/helpers_test.go

```diff
@@ -250,3 +250,10 @@ func checkManifestMatches(t *testing.T, testDir, ver, name string, res ...*regex
 		}
 	}
 }
+
+func mustCreateDeltas(t *testing.T, manifest, statedir string, from, to uint32) {
+	failed, err := CreateDeltas(manifest, statedir, from, to)
+	if err != nil || len(failed) > 0 {
```

Comment:
This seems to be mixing two cases, 'err != nil' means one thing and 'len(failed)>0'.
Probably need two different print statements for the two cases.